docs/flink-getting-started.md

```diff
@@ -687,11 +664,11 @@ FlinkSink.forRowData(input)
 env.execute("Test Iceberg DataStream");
 ```
 
-The iceberg API also allows users to write generic `DataStream<T>` to iceberg table, more example could be found in this [unit test](https://github.com/apache/iceberg/blob/master/flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSink.java).
+The iceberg API also allows users to write generic `DataStream<T>` to iceberg table, more example could be found in this [unit test](https://github.com/apache/iceberg/blob/master/flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSink.java).
 
 ### Overwrite data
 
-To overwrite the data in existing iceberg table dynamically, we could set the `overwrite` flag in FlinkSink builder.
+To overwrite the data in existing iceberg table dynamically, set the `overwrite` flag in FlinkSink builder.
```

Comment:
is this a complete sentence? seems missing noun :) Maybe reverse the order like you did in some other places. `Set the overwrite flag ... to overwrite data in the existing partitions.`?

Reply:
Yes, the sentence doesn't read very easily. I like your suggestion very much. Updated!